.github/workflows/binary-installers.yml

```diff
@@ -0,0 +1,64 @@
+name: Create binary installers
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  pip-compile:
+    strategy:
+      fail-fast: false
+      matrix:
+        output-file:
+          - py3.10-darwin-x86_64-reqs
+          - py3.10-linux-x86_64-cuda-reqs
+          - py3.10-windows-x86_64-cuda-reqs
+        python-version:
+          - '3.9'
```

Comment:
Why?

Reply:
As already written to you in Discord, this is for Debug Reason (maybe you also saw the big fat "DO NOT MERGE" in the first comment ...)